README.md

```diff
@@ -167,6 +167,27 @@ Kiwix-build.py will create several directories:
 If you want to install all those directories elsewhere, you can pass the
 `--working-dir` option to `kiwix-build`:
 
+
+Building an AppImage on Linux
+-----------------------------
+Here are the exact commands (tested on clean Fedora 34x64):
+```
+sudo dnf install virtualenv ninja-build meson patch qmake-qt5 patchelf
+cd ~
+wget https://github.com/kiwix/kiwix-build/archive/refs/heads/master.zip
+unzip master.zip
+cd kiwix-build-master
+virtualenv -p python3 ./
+source bin/activate
+pip install .
+hash -r
+export SKIP_BIG_MEMORY_TEST=1
+kiwix-build --target-platform native_dyn kiwix-desktop
+./scripts/create_kiwix-desktop_appImage.sh
+```
+The result: ~/kiwix-build-master/Kiwix--x86_64.AppImage
+
```

Comment:
There is already a documentation about how install kiwix-buidl in `prerequisite` section. And how to build kiwix-desktop in the `compilation` section.
All this new documentation could be resumed as: `Compile locally kiwix-desktop on native_dyn platform, then run the script create_kiwix-desktop_appImage.sh` (better written obviously)

Reply:
It wasn't obvious to me, as for a new developer on the project. Also, libzim tests failed on virtual machine with limited resources.

In updated PR the commands are visually compressed under the spoiler:

![Screenshot from 2022-03-14 14-24-53](https://user-images.githubusercontent.com/5078434/158124210-0ea19e38-f69d-4920-8873-6666abc82fba.png)

Reply:
The problem is that the information you give is not relative to creating a appimage. The only relevant information (to create a appimage) is that we must run (and how) `create_kiwix-desktop_appImage.sh` once kiwix-desktop is build.

The other information are useful, but are not related to appimage. We should/may keep them, but probably move them elsewhere (compilation of kiwix-desktop, troubleshooting section, ....)